controllers/space/space_controller_test.go

```diff
@@ -0,0 +1,796 @@
+								HasNoFinalizers() // space resource can be deleted by the server now that the finalizer has been removed
+						})
+					})
+				})
+			})
+
+			t.Run("when space was not successfully provisioned", func(t *testing.T) {
+
+				t.Run("because of missing target member cluster", func(t *testing.T) {
+					// given
+					space := spacetest.NewSpace("oddity", basicTier.Name,
+						spacetest.WithoutTargetCluster(),
+						spacetest.WithFinalizer(),
+						spacetest.WithDeletionTimestamp(),
+						spacetest.WithCondition(toolchainv1alpha1.Condition{
+							Type:    toolchainv1alpha1.ConditionReady,
+							Status:  corev1.ConditionFalse,
+							Reason:  toolchainv1alpha1.SpaceProvisioningFailedReason,
+							Message: "missing target member cluster",
+						}),
+					)
+					hostClient := test.NewFakeClient(t, space, basicTier)
+					member1 := NewMemberCluster(t, "member-1", corev1.ConditionTrue)
+					member2 := NewMemberCluster(t, "member-2", corev1.ConditionTrue)
+					ctrl := newReconciler(hostClient, member1, member2)
+
+					// when
+					res, err := ctrl.Reconcile(context.TODO(), reconcile.Request{
+						NamespacedName: types.NamespacedName{
+							Namespace: space.Namespace,
+							Name:      space.Name,
+						},
+					})
+
+					// then
+					require.NoError(t, err)
+					assert.Equal(t, reconcile.Result{Requeue: false}, res) // no requeue neede
+					spacetest.AssertThatSpace(t, test.HostOperatorNs, "oddity", hostClient).
+						Exists().
+						HasNoStatusTargetCluster().
+						HasNoFinalizers() // will allow for deletion of the Space CR
+				})
+
+				t.Run("because of unknown target member cluster", func(t *testing.T) {
+					// given
+					space := spacetest.NewSpace("oddity", basicTier.Name,
+						spacetest.WithTargetCluster("unknown"),
+						spacetest.WithFinalizer(),
+						spacetest.WithDeletionTimestamp(),
+						spacetest.WithCondition(toolchainv1alpha1.Condition{
+							Type:    toolchainv1alpha1.ConditionReady,
+							Status:  corev1.ConditionFalse,
+							Reason:  toolchainv1alpha1.SpaceProvisioningFailedReason,
+							Message: "unknown target member cluster 'unknown'",
+						}),
+					)
+					hostClient := test.NewFakeClient(t, space, basicTier)
+					member1 := NewMemberCluster(t, "member-1", corev1.ConditionTrue)
+					member2 := NewMemberCluster(t, "member-2", corev1.ConditionTrue)
+					ctrl := newReconciler(hostClient, member1, member2)
+
+					// when
+					res, err := ctrl.Reconcile(context.TODO(), reconcile.Request{
+						NamespacedName: types.NamespacedName{
+							Namespace: space.Namespace,
+							Name:      space.Name,
+						},
+					})
+
+					// then
+					require.NoError(t, err)
+					assert.Equal(t, reconcile.Result{Requeue: false}, res) // no requeue neede
+					spacetest.AssertThatSpace(t, test.HostOperatorNs, "oddity", hostClient).
+						Exists().
+						HasNoStatusTargetCluster().
+						HasNoFinalizers() // will allow for deletion of the Space CR
```

Comment:
just commented in the reconcile code - this is an error case

Reply:
see comment above

Reply:
done in ba33697